rhscl-charter.md

```diff
@@ -209,6 +209,8 @@ Daemon is listening on 0.0.0.0 by default.
 
 Data directory (if any) that is expected to be mounted shouldn't be home directory of user, there may be more stuff that we don't want to mount, so in some cases we use data/ subdirectory for data themselves (and VOLUME)
 
+Daemon log errors to `stdout` or `stderr`, so the logs are shown in the `docker log` output. This will probably change as soon as we can use `systemd` in the container and proper `journald` logging.
+
```

Comment:
@praiskup @lnykryn which is not what you want as you will require persistent volume for each container to store logs to and you loose the logs as soon as the container is removed/moved/idled/stopped... In future, enterprise users will want OpenShift to use tools like logstash to collect **all** logs and give it to them in our web UI.

For now, `docker logs` is the way to go to get the logs, so we have to stick with sending them to stdout/stderr.

@lnykryn it is possible to configure systemd which runs inside container to instead of store journal on disk, just stream longs to /dev/stdout?